README.md

```diff
@@ -15,6 +15,10 @@ import { foo } from "bar" // Deno Error
 
 Bundler bundles deno syntax files (url imports, .ts extensions) for the web.
 
+### No Requirements
+- no need for a cache folder in your project. Bundler uses the Deno cache system.
+- no need for a `node_modules` folder and a `package.json` file.
```

Comment:
No need for mentioning node and package.json. This is standard for deno.
```cache``` might be better mentioned under features.